src/screens/home/HomeScreen.tsx

```diff
@@ -35,7 +27,9 @@ export const HomeScreen = ({
   const [launcherContext, setLauncherContext] = useState<LauncherContext>({
     state: 'default'
   })
-  const { launcherClient } = useLauncherClient(launcherContext.value)
+  const { canDisplayLauncher, launcherClient } =
+    useLauncherWrapper(launcherContext)
+  const [concurrentConnector, setConcurrentConnector] = useState('')
```

Comment:
I would set `useState(undefined)` by default

Reply:
Yeah I wanted to stay always a string but can work too

Reply:
✔️